packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -114,35 +114,35 @@ describe('convert-runtime-to-plugin', () => {
       files: [
         {
           input: `../../../inputs/api-routes-python/api/db/[id].py`,
-          output: 'api/db/[id].py',
+          output: '.output/server/pages/api/api/db/[id].py',
```

Comment:
It looks like the `output` prop is now relative to the `workPath` dir.

Should we also make `input` prop relative to the `workPath` dir?

Reply:
Not sure I understand — `output` is not relative, it's the full output path.

Reply:
The `input` property has to be relative to the NFT file.